### DISCUSSION TITLE
Stripe Payment Price

### DISCUSSION THREAD

**Discussion by derevyan** [docusealco/docuseal#332] [Ideas]
It would be fantastic to have the option to set the Stripe payment amount based on a value from another form field value.

**Comment by omohokcoj** [comment]
@derevyan we made it possible to set the payment field price via a formula - you can learn more about how the formula works here:
https://www.docuseal.co/resources/add-formula-fields 

![image](https://github.com/user-attachments/assets/26aa759a-d383-4e58-9416-8857ad59da7c)
